sandbox-open-imis/charts/backend/templates/deployment.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.deployment.name }}
+  namespace: {{ .Values.deployment.namespace }}
```

Comment:
Helm standard is to use {{ .Release.Namespace }} see https://helm.sh/docs/chart_template_guide/builtin_objects/

Reply:
Good point. Thank you for a feedback.
We have other similar charts so before applying best practice need to make a team opinion.
https://govstack-global.atlassian.net/browse/SND-199